### PR TITLE
Add compute algorithm to detailed operations

### PR DESCRIPTION
**Pull request by toji**
Resolves an editorial note by adding a high-level algorithm detailing how compute shader dispatch is handled.

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -14541,7 +14547,63 @@ These operations are encoded within {{GPUComputePassEncoder}} as:
 - {{GPUComputePassEncoder/dispatchWorkgroups()}}
 - {{GPUComputePassEncoder/dispatchWorkgroupsIndirect()}}
 
-<p class="note editorial"><span class=marker>Editorial note:</span> describe the computing algorithm
+The main compute algorithm:
+
+<div algorithm data-timeline=queue>
+    <dfn abstract-op>compute</dfn>(descriptor, drawCall, state)
+
+        **Arguments:**
```
Comment (kainino0x) [inline spec/index.bs:14555]: nit: dedent by one starting here

File: spec/index.bs [hunk 2]
```diff
@@ -14541,7 +14547,63 @@ These operations are encoded within {{GPUComputePassEncoder}} as:
...
+                        |invocation|.|workgroupZ| * |workgroupSize|.`z` + |invocation|.|localZ|<br/>
+                    )</code>.
+                - Set the [=builtin/local_invocation_index=] builtin, if any, to <code>
+                    |invocation|.|localX| + (|invocation|.|localY| * |workgroupSize|.`x`) +
+                    (|invocation|.|localZ| * |workgroupSize|.`x` * |workgroupSize|.`y`)
+                    </code>
+            1. Invoke the compute shader entry point described by |invocation|.|computeStage|.
+
+        Note: Shader invocations have no guaranteed order, and will generally run in parallel according to device
+            capabilities. Developers should not assume that any given invocation or workgroup will complete before any
```
Comment (kainino0x) [inline spec/index.bs:14602]: and dedent by one again starting here

**Review by kainino0x — APPROVED** [review]
LGTM

File: spec/index.bs (hunk 1 above)
Comment (toji) [reply spec/index.bs]: Done

File: spec/index.bs (hunk 2 above)
Comment (toji) [reply spec/index.bs]: Done

**Pull request merged**